controllers/yawol-controller/loadbalancermachine/loadbalancermachine_controller.go

```diff
@@ -35,12 +35,14 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
 )
 
 const (
 	// ServiceFinalizer Name of finalizer for controller4
 	ServiceFinalizer             = "yawol.stackit.cloud/controller4"
 	DefaultRequeueTime           = 10 * time.Millisecond
+	OpenstackRequeueTime         = 1 * time.Minute
```

Comment:
We should use the same duration as the reconcile https://github.com/stackitcloud/yawol/blob/main/internal/helper/loadbalancermachine.go#L27 Maybe also the same `const`